<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Schenky</class>
 <widget class="QMainWindow" name="Schenky">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>614</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1088</width>
    <height>614</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1088</width>
    <height>614</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Schenky</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="keyLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Key signature:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="keySelect">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Choose a key</string>
    </property>
    <item>
     <property name="text">
      <string>C major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>D major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>E major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F# major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C# major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cb major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gb major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Db major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ab major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eb major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bb major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F major</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>E minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F# minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C# minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G# minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>D# minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A# minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ab minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eb minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bb minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G minor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>D minor</string>
     </property>
    </item>
   </widget>
   <widget class="QScrollArea" name="previewScrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>991</width>
      <height>281</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Schenkerian graph preview</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="previewScrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>989</width>
       <height>279</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="previewLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Preview:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>938</x>
      <y>234</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Refresh Schenkerian graph</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoRefreshCheckbox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>238</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Uncheck for better performance</string>
    </property>
    <property name="statusTip">
     <string>Auto-refresh the Schenkerian graph</string>
    </property>
    <property name="text">
     <string>Auto-refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomInButton">
    <property name="geometry">
     <rect>
      <x>1026</x>
      <y>270</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Zoom In</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOutButton">
    <property name="geometry">
     <rect>
      <x>1026</x>
      <y>300</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Zoom Out</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="inputScrollArea">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>911</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="inputScrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>909</width>
       <height>139</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="inputScrollAreaWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>931</width>
        <height>141</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="statusTip">
       <string>Add/remove note</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget { background: white }</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1050</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="inputVerticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QWidget" name="measureInputWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>911</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="measureInputHLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="Line" name="separator_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="trebleInputWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1012</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="trebleInputHLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="Line" name="separator_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="bassInputWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>911</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="bassInputHLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>71</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="labelVerticalLayout">
     <item>
      <widget class="QLabel" name="measureLabel">
       <property name="text">
        <string>Measure:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="trebleLable">
       <property name="text">
        <string>Treble:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bassLabel">
       <property name="text">
        <string>Bass:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="slurButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Select this button then click two notes to form a slur</string>
    </property>
    <property name="text">
     <string>Slur Mode</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="unfoldingButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Select this button then click two notes to form an unfolding</string>
    </property>
    <property name="text">
     <string>Unfolding Mode</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lineButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Select this button then click a sequence of notes to form a line</string>
    </property>
    <property name="text">
     <string>Line Mode</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="urlinieCheckbox">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Urlinie</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="propertiesPanel">
   <property name="maximumSize">
    <size>
     <width>402</width>
     <height>524287</height>
    </size>
   </property>
   <property name="statusTip">
    <string>Properties panel</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="propertiesPanelContents">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>571</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="noteTab">
      <attribute name="title">
       <string>Note</string>
      </attribute>
      <widget class="QTableWidget" name="notePropertiesTable">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>381</width>
         <height>541</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Edit note properties</string>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideRight</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>7</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>125</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>25</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Property</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>ID</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string/>
        </property>
        <property name="flags">
         <set>ItemIsSelectable</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>Note name</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>Accidental</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>Register</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>Slurs to...</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>Unfolds with...</string>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string>Part of lines...</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="slursTab">
      <attribute name="title">
       <string>Slurs</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="unfoldingsTab">
      <attribute name="title">
       <string>Unfoldings</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="linesTab">
      <attribute name="title">
       <string>Lines</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Create a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save file</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
